### PR TITLE
fix version in operation manual

### PR DESCRIPTION
**Pull request by dacolgit**

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.8.3-rc.{height}",
```
Comment (karok2m) [inline version.json:3]: Please revert this change. Main should use `2.8.3-rc.{height}` format.

File: readme.md [hunk 2]
```diff
@@ -1,96 +1,2323 @@
-# Azure Industrial IoT Platform
+[Home](../../readme.md)
```
Comment (karok2m) [inline readme.md:1]: @dacolgit, I believe that the intent was to change operations manual which is `docs/manual/readme.md`, right ? Seems like the updated content from there has been copy-pasted to `readme.md`. Can you please fix this and apply the changes in `docs/manual/readme.md` ?

File: version.json (hunk 1 above)
Comment (karok2m) [reply version.json:3]: BTW, this change can be coming from the fact that your branch is based on `release/2.8.3_rc` instead of `main`.

Comment (dacolgit) [reply version.json:3]: yes
